src/UI/Rest/Controller/Expedition/ExpeditionCommandController.php

```diff
@@ -20,41 +24,47 @@ public function __construct(MessageBusInterface $bus)
     }
 
     /**
+     * @IsGranted("ROLE_MARS_SCIENTIST")
      * @Route("/{id}", methods={"PATCH"})
      */
-    public function startExpedition(Request $request, int $id)
+    public function startExpedition(Request $request, int $id): Response
     {
         $data = json_decode($request->getContent(), true);
 
-        $command = new StartExpeditionCommand(
-            );
-
+        $command = new StartExpeditionCommand($id);
```

Comment:
zmienna command do niczego nie sluzy nalezy ja dac inline w metodzie handle() czyli

$this->handle(new StartExpeditionCommand($id));